GSA/digitalgov.gov
# Component: Refactor kicker deck and title

## Summary
To enhance clarity, a new partial called universal-header.html was created which is designed to handle the display logic for page headers.

### Preview

| New Component | Production |
| ------------- | ---------- |
| [Events page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/refactor-kicker-deck/title/event/2024/06/20/uswds-monthly-call-june-2024/) | [Events page](https://digital.gov/event/2024/06/20/uswds-monthly-call-june-2024/) |
| [Guides](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/refactor-kicker-deck/title/guides/web-analytics-playbook/) | [Guides](https://digital.gov/guides/web-analytics-playbook/) |
| [Resources](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/refactor-kicker-deck/title/topics/accessibility/) | [Resources](https://digital.gov/topics/accessibility/) |



### Solution
A new partial, guide-details.html, was created to separate the logic for guide headers from the main template. This partial is integrated into the existing guide-header.html.
### How To Test
A correct implementation should have no visual differentiation from the production site. Attached are two images showing this. 

1. Run npm start
2. visit any page in the guides/resources section on local server
3. open the same page on the production site and compare the header

---

### Dev Checklist
